conformance/README.md

```diff
@@ -0,0 +1,150 @@
+# Conformance testing for Rust IPFS
+
+This directory contains the scripts used to run interface conformance testing
+for Rust IPFS. It uses `js-ipfsd-ctl`, `ipfs-http-client`, and
+`interface-js-ipfs-core`. This code used to live at
+https://github.com/rs-ipfs/ipfs-rust-conformance but was integrated into
+https://github.com/rs-ipfs/rust-ipfs as working with multiple repositories got
+a bit tedious.
+
+# Usage
+
+Use `setup.sh` to do `npm install` and patch any of the dependencies:
+
+```bash
+$ cargo build -p ipfs-http
+$ cd conformance
+$ ./setup.sh
+```
+
+By default, there is a `http` symlink to `../target/debug/ipfs-http`. You can
+change this to the release binary by modifying the symlink or use a custom
+binary via the environment variable `IPFS_RUST_EXEC`. The default `rust.sh`
+wrapper will record all actions taken by the tests into a single large log
```

Comment:
I think I've seen that you've added `$$` to the `.log` file, so it's no longer a single big file, no?